### DISCUSSION TITLE
Ontological Commoning

### DISCUSSION THREAD

**Discussion by matthewmccarthy11** [life-itself/community#844] [Research]
[Here are the 'initial ideas' on onotlogical commoning ](https://wiki.simongrant.org/doku.php/d:2023-02-22) from @asimong 

A quick definition extracted from the paper: 'ontological commoning refers on the one hand to the ongoing collective process of organic development of an ontology, and on the other hand to the principle that the ontology itself is about the concepts that are recognized in common, and the common language used to talk about those concepts, and the differences between different ontological perspectives.'

Initial questions/comments: 

1: OC is (to put it in different words) a method of finding a shared language/set of concepts (ontology) which organize further activities (particularly when the context is complex/aims to solve a complex problem)
2: Can OC just be seen as 'human self-organization' (in regards to 'establishing some 'community' or 'social system')
3: An example of OC which also might be meaningful: education reform?
4: It does make sense to consider the psychological dimension (more so just agreeing)
5: I am not sure what you mean by 'technical systems'- also, does ontological commoning mostly refer to 'digital' spaces?
6- If someone is born into a particular culture, and more so absorbs, opposed to creates, certain concepts (or activities or processes), and they readily participate in such processes, is that an example of ontological commoning? (assuming that it is)

**Comment by asimong** [comment]
Thanks for opening this thread Matthew @matthewmccarthy11 ! I''m honoured – or should that be “flattered” – that you're taking my concept as the title of a thread! I'll leave this as a thank you comment and make the actual replies separately.

Oh, and cross-back-linking to the topic this emerged from: #839

**Comment by asimong** [comment]
1: _“OC is (to put it in different words) a method of finding a shared language/set of concepts (ontology) which organize further activities (particularly when the context is complex/aims to solve a complex problem)”_

Yes, a method, and also I would say more, an attitude or orientation — I don't see much hope for a method without that internal attitude or orientation — towards seeking a deep understanding of the other's ontology, enquiring into the worldview, mindset and psychological precursors which may have led to the other's ontology, including the trendy topics like trauma awareness, and my own _idée fixe_ of healing narrative.

To me, this vitally involves practices like deep listening, which I see as much more effectively carried out collectively than individually, though some individuals are either gifted of practiced in it. Also a fundamental openness and humility, expressed perhaps in the recognition that one's own perspective is just one among many equally plausible or valuable or useful other perspectives.

**Comment by asimong** [comment]
2: _“Can OC just be seen as 'human self-organization' (in regards to 'establishing some 'community' or 'social system')”_

The word “just” brings up flashing amber warning lights for me!

Yes, I think it could be seen through the lens of human self-organisation. How complex, rich and diverse is that! So, I would say, please no need for the word “just”

It could also be seen through the lenses of organisational design, development, change, etc. That is, it could be facilitated by an outsider, and this might be necessary if the attitudes and orientations mentioned above in (1.) are lacking.

**Comment by asimong** [comment]
3: _“An example of OC which also might be meaningful: education reform?”_

The tricky word here seems to be “reform”, as that very word, contrasted for example with “revolution”, has its own agenda and worldview. If I were to look at OC around what needs to change around the education system, OC would need, among other things, to look into the belief systems attached to using the terms “reform” or “revolution” (or any other change word of choice)

The distinction I'd like to make here is in the depth (oops!) of change — the Overton window, perhaps. You could have a restricted OC exercise between reformists of different flavours. They would perhaps agree on a common set of undisputed (among them) assumptions, aligned with the “reform” mindset, and perhaps that would help them feel enough common ground then to talk about their different reformist agendas.

An OC process including reformists and revolutionaries would need to be much more radical, with fewer shared starting assumptions. This seems to me both more challenging and more potentially generative, as the initial mismatch between ontologies would likely be greater.

**Comment by asimong** [comment]
4: _“It does make sense to consider the psychological dimension (more so just agreeing)”_

Thank you! To me, OC starts when the psychological dimension is recognised as significant. Thus, more than making sense, I would say it is essential ­— of the essence.

**Comment by asimong** [comment]
5: _“I am not sure what you mean by 'technical systems'- also, does ontological commoning mostly refer to 'digital' spaces?”_

Second part of this: certainly not: OC refers very much to human ontologies independent of technology.

My main point about technical systems is to do with the difficulties in establishing technical interoperability in cases where there is no ontological common ground. Dealing with IT brings out the need to make things explicit that we have been able to get away with as implicit in normal human communication.  A consideration of IT systems bring to a head the need for OC as preparatory to the implementation of systems that are designed to serve “the commons” in any meaningful way.

**Comment by asimong** [comment]
6: _“If someone is born into a particular culture, and more so absorbs, opposed to creates, certain concepts (or activities or processes), and they readily participate in such processes, is that an example of ontological commoning? (assuming that it is)”_

Thanks for bringing this up, as I would be inclined to say, no. There is another view possible, as represented by the “Psycommons” concept of Denis Postle. And you could say that the cultural ontology is a kind of commons — discuss, with reference to Ostrom? One challenge is that culture is not governed by its participants through any explicit process. Would people collectively have designed and consented to a social media culture that algorithmically promotes strife?

But what an absorbent process doesn't get is what I see as a characteristic, essential **process** of OC, which I do see as something that needs to be conscious. Hat-tip here to @rufuspollock  and the “awakened” thread of being. If it's not conscious, I would just lump it in with “enculturation” as has been written about much, and I know very little of that literature.

**Reply by asimong** [reply]
come on dear researchers, I'd love to see more responses here!

**Reply by matthewmccarthy11** [reply]
Lol

**Reply by matthewmccarthy11** [reply]
Will be sending over some replies soon

**Comment by rufuspollock** [comment]
To note i've read the thread (but not the full paper yet)

- Ontology as "a shared set of key ideas and concepts" in a domain is reasonable if explicitly qualified as "ontology for domain X". Obviously we use ontology frequently at Life Itself to mean "the being of human beings" or an expansive version of psychology including worldview, psychology etc.
- Getting agreement - or more importantly getting deeply in sync - on terminology in a domain is a big deal and very valuable (i think of landmark here and especially https://www.landmarkworldwide.com/masterycourse

**Reply by asimong** [reply]
Indeed, @rufuspollock , 100% agreed on the value of cohering terminology. But could you please explain (to me at least, and it wouldn't have to be in writing) the Life Itself usage you mention here? Surely when we say "ontology" we are talking about talking – words – aren't we? Not just some presumed “truth” about the being of being human? When I talk about human ontologies, my intention is to refer to what each individual makes of their world and their life. And, because the "task" of life is (to say the least) complex (teetering on the brink of chaos sometimes!) surely each individual's ontology will be different?

**Reply by rufuspollock** [reply]
Hmm, i think i am using the term ontology in those two sense which could be ambiguous e.g.

"We live in an ontological-social paradigm": here ontology is meaning the overall frame in which we see and act in the world ie. the set of views and values and overall psychology that gives our being and acting in the world.

At other times what i mean by ontology is the study of our "being in the world" - some combination of psychology, spirituality, philosophy and more. i.e. the investigation into the question: "What is nature of being for human beings".

**Reply by asimong** [reply]
>"We live in an ontological-social paradigm": here ontology is meaning the overall frame in which we see and act in the world ie. the set of views and values and overall psychology that gives our being and acting in the world.

however what I'm saying is there is no one overall frame, it varies from person to person. So for me, it's "a set" not "the set". Of course that may be what you meant anyway, just wanted to be clear. And the process of ontological commoning is all about bringing all that to the surface, so that the similarities and differences can be talked through.

**Reply by asimong** [reply]
>At other times what i mean by ontology is the study of our "being in the world" - some combination of psychology, spirituality, philosophy and more. i.e. the investigation into the question: "What is nature of being for human beings".

Sure, ontology as a discipline. Is anyone seriously claiming that there will be, or even can be, general agreement on these matters? I'm with you so long as you anticipate many possibilities.

**Reply by rufuspollock** [reply]
I believe there can be agreement on a bunch of these matters, yes. I've strong believer in that there is "ultimate" truth though it is one that transcends language and we can never fully know it.

Perhaps the best analogy here is that people can agree on the direction up the mountain even if we don't agree on what the "top" of the mountain is.

**Comment by matthewmccarthy11** [comment]
Re-entering the chat here: I checked out Denis Postle, and found a [summary of the psycommons here](https://www.bollier.org/blog/denis-postle-psycommons) from a guy named David Bollier. I looked more into his podcast 'frontiers of commoning' which is really great. [There is a new episode out today!](https://david-bollier.simplecast.com/)

I think this idea of psy commons is really interesting, and maybe resembles a few other ideas which maybe carry a different worldview, although framing it as the psycommons might be helpful. I only briefly read the introduction from Bollier, and he gives the following definition:

"the psyCommons is the realm of the informal, the customary and the local – the social spaces in our lives that are largely exempt from bureaucratic or legal control, the spaces where people can negotiate their own shared understandings of intersubjective reality"

He also relates it to the 'vernacular' (Illich), which refers to "a sensibility and rootedness…in which local life has been conducted throughout most of history and even today in a significant proportion of subsistence-and communitarian-oriented communities...[they are] places and spaces where people are struggling to achieve regeneration and social restoration against the forces of economic globalization,” 

Considering these concepts or considering 'cultural ontology' (not sure if I'm using the word right here) it seems like an interesting way to contextualize 'ontological commoning', and wanted to get your opinion on how 'ontological commoning' depends to some (or a large) degree on the world/culture that it occurs in and the 'common structures/values' of how 'ontolgoical commoning' takes place, and just wanted to share a few comments as it relates to this.

In my experience, 'ontological commoning' in a lot of contemporary spaces can be not very meaningful across a few different dimensions, and I would maybe even suggest that 'ontological commoning' doesn't *really happen* in some cases where it appears to be happening. A question here: is  ontological commoning happening, if the people who are doing the commoning are regurgitating certain heirarchical values? (a bit vague, but feel free to answer in whatever way seems fit)

One might say there is a 'contemporary (or modern) orientation' to ontological commoning, and I think some interesting examples to consider here might be 'community town hall meetings'. Would you consider this to be an example of ontological commoning? (or is it mainly about the 'intitial concepts/views that organize these activities) To run with the example here, these meetings have an 'open discussion' component at times, which I would maybe consider as OC, but want to point out: in a town/city of 50,000+ people, there might be 10-15 people (tops!) who show up , and represent a small part of the population, less than .1%, and if these 10 people talk at this meeting, it seems like community leaders would be thrilled at thus number of participants. 

And one element here of 'ontological commining' which has this 'modern orientation': that it has a 'spectacular' element, and I use the word in reference to it being a 'spectacle' of inclusion- (also using spectacular to hold the irony of how 'unspectacular' or even uncomfortable, it can be). Another example here might be certain DEI initiatives, which aim to have the spectacle of ontological commoning and 'the process of ontological commoning' is the 'spectacle of the ontological commoning', at least to some degree. Not sure if this makes sense, maybe it can be worded better. (Also worth pointing out: atleast I guess there is a sense of trying to OC, even if it has this element) 

Another point here is that in the 'contemporary sense, there must be a purpose as it relates to ontological commoning- and of course it makes sense to consider this in the overall scheme of what it is, but this 'purpose' or 'pragmatic mindset' is often viewed as the 'voice of reason' in ontological commoning. 

Lots more to say about the psycommons and related things, but in sum just wanted to get your feedback on 'ontological commoning' (generally speaking) in our current world as it might relate to certain 'modern' orientations/tendencies, which include (maybe) a 'spectacle' element, and a 'pragmatic' element. Also, maybe getting some examples as it relates to OC to see how it plays out (or doesn't) in our world, like for example town hall meetings or DEI initiatives. Also, successful examples which make sense, and I didn't include any here. (Clarifying here: not suggesting that DEI initiatives are bad, but they can be done for the purpose of box-checking so that one appears to be inclusive)
